### ISSUE TITLE
exchange_rate: internal server error

### ISSUE THREAD

**Issue by lasers** [ultrabug/py3status#1569]
This module hasn't been working for months. See if we can find a different server or something.
```json
{
    "name":"Internal Server Error",
    "message":"An internal server error occurred.",
    "code":0,
    "status":500
}
````
```
$ python exchange_rate.py --term 
$? £? ¥?
$? £? ¥?
$? £? ¥?
```

**Comment by tobes** [comment]
I've found a decent feed I'll fix it and will make a PR.

**Comment by lasers** [comment]
Closed via #1655.
